cli/golem-templates/templates/rust/rust-app-common/Cargo.toml._

```diff
@@ -7,8 +7,9 @@ opt-level = "s"
 lto = true
 
 [workspace.dependencies]
-golem-rust = { version = "1.8.1", features = ["export_load_snapshot", "export_save_snapshot", "export_oplog_processor"] }
+golem-rust = { path = "../golem/sdks/rust/golem-rust", features = ["export_load_snapshot", "export_save_snapshot", "export_golem_agentic"] }
+golem-wasm = {version = "1.3.2-dev.2", default-features = false, features = ["stub"]}
```

Comment:
This should not be explicitly here, because `golem-rust` brings it in, and reexports the whole thing as a submodule, exactly to avoid having to depend on it like this